kuma/javascript/src/payments/__tests__/feedback-form.test.jsx

```diff
@@ -0,0 +1,87 @@
+//@flow
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
```

Comment:
Awesome!

Reply:
Thank you! I like how this test turned out and makes me appreciate `testing-library/react` even more